### PR TITLE
Update requirements for Xamarin

### PR DESCRIPTION
**Pull request by frederikprijck**
Bumped the requirements:

- Include VS2019
- Drop the version number from Xamarin (it comes installed through VS Installer, you don't have to google for a specific version and I am also not sure what Xamarin for VS 4.5 is about, I have Xamarin 16.8 installed).
- Bump SDK versions

### CODE REVIEW

File: articles/quickstart/native/xamarin/index.yml [hunk 1]
```diff
@@ -25,10 +25,10 @@ github:
   org: auth0-samples
   repo: auth0-xamarin-oidc-samples
 requirements:
-  - Visual Studio 2017 or Visual Studio for Mac
-  - Xamarin for Visual Studio 4.5
+  - Visual Studio 2017, 2019 or Visual Studio for Mac
```
Comment (stevehobbsdev) [inline articles/quickstart/native/xamarin/index.yml]: > Visual Studio 2017, 2019

Do you think we'd be better off saying something like "Visual Studio 2017 or later" here (as well as VS for Mac)? What happens when VS 2021 comes out?

**Review by stevehobbsdev — COMMENTED** [review]
🎉

File: articles/quickstart/native/xamarin/index.yml (hunk 1 above)
Comment (frederikprijck) [reply articles/quickstart/native/xamarin/index.yml]: `Visual Studio 2017 or later or Visual Studio for Mac` reads a bit weird I think. What do u think about `Visual Studio 2017+ or Visual Studio for Mac`

Comment (stevehobbsdev) [reply articles/quickstart/native/xamarin/index.yml]: That works 👍

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**